base/Inn.ts

```diff
@@ -0,0 +1,33 @@
+export class Inn {
+    public inn_id: string;
+    public avatar_key: string;
+    public cost: number;
+    public welcome_message: string;
+    public price_message: string;
+    public cancel_message: string;
+    public stay_message: string;
+    public goodbye_message: string;
+    public not_enough_coins: string;
+
+    constructor(
+        inn_id: string,
+        avatar_key: string,
+        cost: number,
+        welcome_message: string,
+        price_message: string,
+        cancel_message: string,
+        stay_message: string,
+        goodbye_message: string,
+        not_enough_coins: string
+    ) {
+        this.inn_id = inn_id;
+        this.avatar_key = avatar_key;
+        this.cost = cost;
+        this.welcome_message = welcome_message;
+        this.price_message = price_message;
+        this.cancel_message = cancel_message;
+        this.stay_message = stay_message;
+        this.goodbye_message = goodbye_message;
+        this.not_enough_coins = not_enough_coins;
```

Comment:
I think that would be better if we set all message variables inside a `message` variable:
```javascript
constructor(
    inn_id: string,
    avatar_key: string,
    cost: number,
    messages: {
        welcome_message: string,
        price_message: string,
        cancel_message: string,
        stay_message: string,
        goodbye_message: string,
        not_enough_coins: string
    }
) {
    this.inn_id = inn_id;
    this.avatar_key = avatar_key;
    this.cost = cost;
    this.messages = {
        welcome_message: messages.welcome_message,
        price_message: messages.price_message,
        cancel_message: messages.cancel_message,
        stay_message: messages.stay_message,
        goodbye_message: messages.goodbye_message,
        not_enough_coins: messages.not_enough_coins,
    }
}
```